dataactbroker/handlers/fileHandler.py

```diff
@@ -108,6 +109,9 @@ def submit(self,name,CreateCredentials):
                 # If filetype not included in request, and this is an update to an existing submission, skip it
                 if not safeDictionary.exists(fileType):
                     if existingSubmission:
+                        submission = self.jobManager.getSubmissionById(submissionId)
```

Comment:
@bsweger I did that because that loop has a check if this is a valid file type before checking for the existing submission. Just made sense to put that update right before the continue within that if block.